gofsutil_mount_unix.go

```diff
@@ -114,7 +114,7 @@ func (fs *FS) getDevMounts(ctx context.Context, dev string) ([]Info, error) {
 
 	var mountInfos []Info
 	for _, m := range allMnts {
-		if m.Device == dev {
+		if m.Device == dev || (m.Device == "devtmpfs" && m.Source == dev) {
```

Comment:
Hi @codenrhoden,

Hmm. This is the exact type of reason for which I created the `ScanEntryFunc`. While the default logic may work for you, it may not for others. I wonder if we shouldn't implement a `ScanEntry` option for matching in `GetDevMounts`?

Reply:
I guess I see your point. But in some ways, if someone provides a non-default `ScanEntryFunc`, all bets are off as to what fields like `m.Device` and `m.Source` are populated with. To the point that `GetDevMounts()` can't really be relied on, no?  Seems like the better option might be to remove `GetDevMounts` from `gofsutil` entirely, and have that be implemented by users themselves if needed -- because if you provide a custom `ScanEntry`, then you are the only person who would know how to implement `GetDevMounts()`.

Reply:
Hi @codenrhoden,

Yes and no. I'm suggesting that we create two `ScanEntryFunc` fields. One for `GetMounts` and one for `GetDevMounts`. The two default functions will be related, and if someone wants to use custom versions for either than they are free to do so.

Reply:
Roger. I leave it up to you. That's more than I want to take on.  No rush -- as I'm able to move on w/o any change. It's just that instead of using `GetDevMounts` right now, I am calling `GetMounts` and then ranging through those results to get all the Dev mounts -- essentially re-implement `GetDevMounts` locally.